### PR TITLE
Improve openfisca serve documentation to make parameter name change inconsistency more obvious

### PR DESCRIPTION
**Pull request by jvalduvieco**
#### Breaking changes

None

#### New features

- Better documentation for openfisca serve mainly raising awarness about inconsistency between parameter names in cmd line and config file.
- gunicorn parameters can be used in openfisca serve. Make this _VERY_ obvious.

#### Deprecations

- None

#### Technical changes

- None

### CODE REVIEW

**Comment by Anna-Livia** [comment]
Seems good for me :) @sandcha  ?

File: doc/source/openfisca_serve.rst [hunk 1]
```diff
@@ -42,14 +48,16 @@ Serving reforms
...
 
 **config.py:**
 
 .. code-block:: py
 
   port = 4000
   workers = 4
-  country-package = 'openfisca_france'
+  bind = 0.0.0.0:4000
+  country_package = 'openfisca_france'
```
Comment (sandcha) [inline doc/source/openfisca_serve.rst]: 👍 👍

File: doc/source/openfisca_serve.rst [hunk 2]
```diff
@@ -42,14 +48,16 @@ Serving reforms
...
+You can setup `openfisca serve` using a configuration file. Be careful as parameters with a '-' in their name change to an '_' when used from the config file.
 
 **config.py:**
 
 .. code-block:: py
 
   port = 4000
   workers = 4
-  country-package = 'openfisca_france'
+  bind = 0.0.0.0:4000
```
Comment (sandcha) [inline doc/source/openfisca_serve.rst]: Update to `bind = '0.0.0.0:' + str(port)` as bind takes a string value and `port` variable is defined above?

> `config.py` is an extension of [gunicorn configuration file](http://docs.gunicorn.org/en/stable/configure.html#configuration-file). It manages openfisca specific configuration like `port`. `port` value will also update `bind` value (openfisca configuration has higher priority here than gunicorn's configuration).

**Comment by fpagnoux** [comment]
It seems that all the comments have been taken into account. Is this PR good to merge ?

**Pull request merged**